source/material_model/rheology/diffusion_creep.cc

```diff
@@ -270,7 +311,7 @@ namespace aspect
         activation_volumes_diffusion = Utilities::MapParsing::parse_map_to_double_array(prm.get("Activation volumes for diffusion creep"),
                                                                                         options);
 
-        grain_size = prm.get_double("Grain size");
+        fixed_grain_size = prm.get_double("Grain size");
```

Comment:
Could you extend the documentation of this parameter in `declare_parameters` above to say that this grain size is only being used if the active material model does not provide its own (possibly changing grain size) when calling this rheology?